netket/machine/torch.py

```diff
@@ -17,7 +17,8 @@ def __init__(self, module, hilbert):
         self._module = _torch.jit.load(module) if isinstance(module, str) else module
         self._module.double()
         self._n_par = _get_number_parameters(self._module)
-
+        self._parameters = list(self._module.parameters())
+        self.n_visible = hilbert.size
```

Comment:
this is unecessary, I think that the call to 
```python
super().__init__(hilbert)
```
already does it

Reply:
No, as far I understand, `self.n_visible = hilbert.size` is necessary, otherwise the test_nvisible test fails